_posts/2020-06-13-IrishGen-RDF-Linked-Data-Semantic-Web.md

```diff
@@ -0,0 +1,398 @@
+_item_ which holds the individual (for the item structure of the
+genealogies, see Holmberg, '[Towards a Relative Chronology of the
+Milesian Genealogical
+Scheme](https://dash.harvard.edu/handle/1/37945002)', pp. 17-18).  The
+data, of course, is inside the file.  This is useful when searching as
+many systems will give the full URL for what they find and the user
+can track down the exact item that the information appears in.
+
+With the formalities of an individual's URL out of the way, the reader
+can see the above Notation3 would be very cumbersome for humans to
+either create or read.  Thus, Notation3 is not often used in practice.
+A more human readable format is Turtle of which TRiG, already
+mentioned, is a slight extension.  For example, the above can be
+written:
+
+```turtle
+@base <http://example.com/LL/dal_corpri_arad.trig> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+
+<http://example.com/LL> {
+  <#Flaithbertach> a foaf:Person 
+}
+```
+
+This is a much condensed and in a way easier to read version of the
+above.  Turtle (and by extension TRiG) condenses the format by moving
+common elements to the beginning of the file with shortened prefixes
+to be reused elsewhere and giving sensible defaults for very commonly
+used elements.  In the above, the URL which defines Flaithbertach to
+the system is pulled mostly into the `@base` which means that any URL
+fragment will be given this full URL when read by the system.  The URL
+`<http://www.w3.org/1999/02/22-rdf-syntax-ns#type> ` is merely given a
+default expression as `a` due to the fact that it is very common.
+`foaf:` is declared as a shortened prefix at the top of the file and
+reused.  In IrishGen, there are many people as they make up the
+majority of the data so keeping it short is good practice.
+
+
+## Ontologies and IrishGen
+
+At this point a reader may wonder where predicates come from, which is
+an entirely reasonable subject to consider at this point.  Predicates
+may come from one of two sources.  The first is the RDF specification
+itself which defines a few, general purpose predicates.  The other,
+and far more complex, source is two specifications: [RDF
+Schema](https://www.w3.org/TR/rdf-schema/) (RDFS) and the Web Ontology
+Language (OWL), which has two versions (see [Web Ontology Language
+Reference](https://www.w3.org/TR/owl-ref/) for version one and [OWL 2
+Web Ontology Language Document Overview (Second
+Edition)](https://www.w3.org/TR/owl2-overview/) for version two).  For
+all practical purposes, OWL 2 is the _de facto_ source for all
+predicates in IrishGen even if they are declared with an RDFS URL.
+
+The curators have modified various publically avaliable ontologies and
+created a few of their own to more accurately model the situation in
+the medieval Irish genealogical source material.  For instance,
+ the curators created the idea of `PopulationGroup` which are fairly
+common entities in the geneaologies and the ability of persons to be
```

Comment:
The grammar here is wrankling with me (plural verb, despite "idea" and "PopulationGroup" both being singular). It is of no great importance but perhaps change it to "...the idea of `PopulationGroup`. Population Groups are ..."?